engine/redis-helper.js

```diff
@@ -0,0 +1,24 @@
+import redis from 'redis';
+
+function getClient(dbTestNumber) {
+  const client = redis.createClient(process.env.REDISCLOUD_URL, { no_ready_check: true });
+  return new Promise((resolve, reject) => {
+    if (dbTestNumber) {
+      client.select(dbTestNumber, (err) => {
+        if (err) {
+          reject(err);
```

Comment:
Calling `reject` doesn't return control flow. We'll continue to the `resolve` call below, though I'm not sure what that will actually do.